courseProject/src/main/java/com/neoflex/CourseProject/service/ImplPreScoringService.java

```diff
@@ -0,0 +1,74 @@
+package com.neoflex.CourseProject.service;
+
+import com.neoflex.CourseProject.dto.LoanOfferDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+@Service
+@Slf4j
+@PropertySource("classpath:application.properties")
+public class ImplPreScoringService implements PreScoring {
+
+    private static final int POINTS_FOR_INSURANCE = 3;
+    private static final int POINTS_FOR_SALARY_CLIENT = 1;
+    private static final int INSURANCE_PRICE = 100000;
+
+    @Value("${ScoringAndPreScoring.rate}")
+     private int defaultRate;
+
+     private List<LoanOfferDto> creatingVariationsOfLoan(){
+        List<LoanOfferDto> variations = new ArrayList<>();
+        variations.add(LoanOfferDto.builder().isInsuranceEnabled(Boolean.FALSE).isSalaryClient(Boolean.FALSE).build());
+        variations.add(LoanOfferDto.builder().isInsuranceEnabled(Boolean.FALSE).isSalaryClient(Boolean.TRUE).build());
+        variations.add(LoanOfferDto.builder().isInsuranceEnabled(Boolean.TRUE).isSalaryClient(Boolean.FALSE).build());
+        variations.add(LoanOfferDto.builder().isInsuranceEnabled(Boolean.TRUE).isSalaryClient(Boolean.TRUE).build());
+        log.info("creatingVariationsOfLoan result: " + variations.toString());
```

Comment:
Для того, чтобы избавиться от всех .toString() у классов, которые ты добавляешь в логи, воспользуйся встроенным форматированием данного метода с помощью такой конструкции:
```suggestion
        log.info("creatingVariationsOfLoan result: {}", variations);
```

Reply:
Вроде исправлено(возможно где-то забыл)
![image](https://github.com/user-attachments/assets/6fdffc30-3607-4d3f-a0d6-604b1be34355)